exercises/numbers/isOdd/isOdd.js

```diff
@@ -15,25 +15,27 @@
  */
 function isOdd(num) {
   // Hint: See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder
-  return _____;
+  return num % 2 === 1;
 }
 
 if (require.main === module) {
   let oddInputs = [1, 19, 99, 767];
 
   console.log('Running sanity checks for isOdd');
 
-  console.log(isOdd(0) === false);
+  console.log(isOdd(0));
 
   // These should all be odd
   for (let num of oddInputs) {
-    console.log(isOdd(num) === true);
+    console.log(isOdd(num));
   }
 
   // If num is odd then num + 1 is even
   for (let num of oddInputs) {
-    console.log(isOdd(num + 1) === false);
+    console.log(isOdd(num + 1));
   }
 }
 
 module.exports = isOdd;
+
+//this isnt working
```

Comment:
So, this _is_ working, but you're misinterpreting the output.

`isOdd(num)` returns `true` or `false` depending on whether `num` is odd. You're using `console.log` to print the return value of `isOdd` directly, so you'd expect `console.log(isOdd(17))` to print `false` since `17` is odd.

The idea of the sanity checks is to give you a way of expressing your expectations in code so that when the computer runs those sanity checks it's easy to see whether any expectations aren't being met.

Since your _expectation_ is that `isOdd(17)` returns `false`, you want something like:

```js
console.log(isOdd(17) === false);
```

If `isOdd(17)` returns `false` (as we expect) then `isOdd(17) === false` evaluates to `true` and we print that.

So you were probably seeing `console.log(isOdd(17) === false);` print `true` and interpreting it as `isOdd(17)` returning `true`. But that line would only print `true` if `isOdd(17)` returned `false`!

If that doesn't make sense — it's kind of hard to write out — just focus on the logic of the function itself. Have it print out whatever makes sense to you and we can adjust from there.